clang/test/dpct/dpct-intercept-build/Makefile

```diff
@@ -8,7 +8,7 @@ $(program): main.cpp test
 
 
 test: 
-	$(CC) -c kernels/test.cu -o test.o 
+	$(CC) -c kernels/test.cu -o test.o -arch=all -arch=native -arch=all-major --gpu-architecture=all --gpu-architecture=native --gpu-architecture=all-major
```

Comment:
Is the case that "a semicolon-separated list of integers, each optionally followed by '-real' or '-virtual'" be covered in this PR?

Reply:
Sync-ed with Chenwei. We haven't seen the usage of this. So currently don't  support it.